### PR TITLE
added category to the model and json file

### PR DESCRIPTION
**Pull request by olhadanylchenko**

### CODE REVIEW

**Comment by olhadanylchenko** [comment]
@karlkristopher @bl00dymarie please review and merge

File: routes/plants.js [hunk 1]
```diff
@@ -1,49 +1,50 @@
...
+router.get("/add", (req, res) => {
+  res.render("plants/add");
 });
 
-router.get('/', (req, res, next) => {
+router.get("/", (req, res, next) => {
   Plant.find()
-    .then(plants => {
-      res.render('plants/index', { plantsList: plants });
+    .then((plants) => {
```
Comment (bl00dymarie) [inline routes/plants.js:12]: oh god this is why the plants index route wasn't working, YAS, thanks for catching it.

File: routes/plants.js [hunk 2]
```diff
@@ -1,49 +1,50 @@
...
+      res.render("plants/index", { plantsList: plants });
     })
-    .catch(err => {
+    .catch((err) => {
```
Comment (bl00dymarie) [inline routes/plants.js:15]: and same here!

**Comment by bl00dymarie** [comment]
Sweet! I however don't see my plant after adding one, is this expected?

image

**Pull request merged**